.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: QA DEPLOY
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: authenticate npm
+      run: npm set //npm.pkg.github.com/:_authToken ${{ secrets.PACKAGING_TOKEN }}
+
+    - run: git config --global user.name kaiomagalhaes
+    - run: git config --global user.email me@kaiomagalhaes.com
```

Comment:
Is there any chance we can handle those values through env variables? It's personal info